README.md

```diff
@@ -13,7 +13,7 @@ This bundle integrates with [league/oauth2-client](http://oauth2-client.thephple
 
 ## Requirements
 
-* PHP 5.5.9 or higher
+* PHP 7.1 or higher
```

Comment:
Let's be more accurate here, from the [composer.json](https://github.com/knpuniversity/oauth2-client-bundle/blob/master/composer.json) I see `^7.1.3`